content/participants/ttjc@ravisaxena23/index.md

```diff
@@ -30,18 +30,14 @@ It would be good if someone tells me how to do auth for apps.
 
 ### How to contact me?
 
-- Twitter
-- LinkedIn
-- E-mail
-- Call
+- LinkedIn https://www.linkedin.com/in/ravisaxena23/
+- E-mail ravisaxena230599@gmail.com
+- Call 9643019798
 
 ### Projects
 
 not a lot of projects are done. I will keep updating it with real links when done.
 
-My projects: _github link_ https://github.com/ravisaxena23/technojam-frontend
-             _github link_ https://github.com/ravisaxena23/crypto-ciphers
-
 
 #### Binder: tinder for books
```

Comment:
Remove this project (Binder) and the blog below it. These are just meant to show you the format. You need to add your own projects and blog here.

Reply:
should i add this thing in the base readme? it may help others although i have not added it now

Reply:
Its clearly mentioned in the registration video to keep this place empty.